 g_turret.c | 4 ++--
 p_client.c | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/g_turret.c b/g_turret.c
index 49dd041..baaf048 100644
--- a/g_turret.c
+++ b/g_turret.c
@@ -252,7 +252,7 @@ Must NOT be on the team with the rest of the turret parts.
 Instead it must target the turret_breach.
 */
 
-void infantry_die (edict_t *self, edict_t *inflictor, edict_t *attacker, int damage);
+void infantry_die (edict_t *self, edict_t *inflictor, edict_t *attacker, int damage, vec3_t point);
 void infantry_stand (edict_t *self);
 void monster_use (edict_t *self, edict_t *other, edict_t *activator);
 
@@ -273,7 +273,7 @@ void turret_driver_die (edict_t *self, edict_t *inflictor, edict_t *attacker, in
 	self->target_ent->owner = NULL;
 	self->target_ent->teammaster->owner = NULL;
 
-	infantry_die (self, inflictor, attacker, damage);
+	infantry_die (self, inflictor, attacker, damage, point);
 }
 
 qboolean FindTarget (edict_t *self);
diff --git a/p_client.c b/p_client.c
index 4061830..5180e93 100644
--- a/p_client.c
+++ b/p_client.c
@@ -153,7 +153,7 @@ void SP_info_player_coop(edict_t *self)
 The deathmatch intermission point will be at one of these
 Use 'angles' instead of 'angle', so you can set pitch or roll as well as yaw.  'pitch yaw roll'
 */
-void SP_info_player_intermission(void)
+void SP_info_player_intermission(edict_t *self)
 {
 }
 
