 src/baseq2/p_client.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/baseq2/p_client.c b/src/baseq2/p_client.c
index 473a148..908418e 100644
--- a/src/baseq2/p_client.c
+++ b/src/baseq2/p_client.c
@@ -158,7 +158,7 @@ void SP_info_player_coop(edict_t *self)
 The deathmatch intermission point will be at one of these
 Use 'angles' instead of 'angle', so you can set pitch or roll as well as yaw.  'pitch yaw roll'
 */
-void SP_info_player_intermission(void)
+void SP_info_player_intermission(edict_t *self)
 {
 }
 
